README.md

```diff
@@ -393,73 +389,6 @@ Please direct general questions and help requests to our [forum](https://discuss
 
 If you believe you've found a bug, check the [issues list](https://github.com/ipfs/kubo/issues) and, if you don't see your problem there, either come talk to us on [Matrix chat](https://docs.ipfs.io/community/chat/), or file an issue of your own!
 
```

Comment:
nit: I think we should have this table in the docs – filled https://github.com/ipfs/ipfs-docs/issues/1208 to update https://docs.ipfs.io/reference/go/api/ page, for now let's just link to it.

```suggestion

## Packages

See [IPFS in GO](https://docs.ipfs.io/reference/go/api/) documentation.

```